test/test_python_dispatch.py

```diff
@@ -460,6 +460,14 @@ def test_nested_enable_python_mode(self) -> None:
                 with enable_python_mode(LoggingTensor):
                     pass
 
+    def test_tolist_numpy_with_python_mode(self) -> None:
+        x = LoggingTensor(torch.tensor([2.0, 3.0], requires_grad=True))
+        with self.assertRaisesRegex(RuntimeError, "is not supported for tensor subclasses."):
+            x.tolist()
+            x.numpy() 
```

Comment:
You need to separate this one in a different context manager, otherwise, it is never tested.